tutorials/config/eum21.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2016-2021 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ReFrame CSCS settings
+#
+
+import reframe.utility.osext as osext
+
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'reframe',
+            'descr': 'Reframe tutorial',
+            'hostnames': [
+                'reframe'
+            ],
+            'modules_system': 'lmod',
+            'resourcesdir': '/apps/reframe/reframe-resources/',
+            'partitions': [
+                {
+                    'name': 'login',
+                    'scheduler': 'local',
+                    'environs': [
+                        'builtin',
+                        'builtin-gcc',
+                        'PrgEnv-gnu',
+                        'PrgEnv-gnu-nompi'
```

Comment:
No Cray notation! You should use the tutorial config as a baseline for this config.